jekyll/_cci2/gitlab-integration.adoc

```diff
@@ -194,6 +256,30 @@ Organization administrator as well as organization contributor roles are current
 +
 If they do not currently have a CircleCI account, they will need to sign up. If they already have a CircleCI account, they are added to the organization, and if they are logged in, they will see the organization as an option in the organization switcher in the top left corner of the web app.
 
+[#organization-settings-integrations]
+=== Integrations (GitLab Self-managed only)
+
+For GitLab Self-managed organizations, you may connect additional self-managed instances to be integrated with your organization. 
+
+. Navigate to **Integrations** within **Organization Settings** to add a new instance. 
+
+. You will need to enter the instance URL, and the SSH host keys if you have not already done so, as described in the xref:#sign-up[Sign up] section above. You have successfully set up the integration when you see a "Known hosts file verified" message.
```

Comment:
**nit/non-blocking** We don't currently have this, but will be adding it soon. At the moment, we only have the option to add a new instance